src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator/ArpGenerator.php

```diff
@@ -44,7 +44,7 @@ public function build(ManageEntity $entity): array
         $attributes = [];
         $entityAttributes = $entity->getAttributes();
         foreach ($this->attributeRepository->findAll() as $definition) {
-            $urns = $definition->getUrns();
+            $urns = $definition->urns;
```

Comment:
Should this change be in this PR? I'm not going to reject your PR over this. But please be aware of what you include in your commits.

Reply:
Almost there! You forgot to add the new service methods (getUrns && getEntityMergeAttributes) to the `AttributeServiceInterface`